library/ecp.c

```diff
@@ -1446,7 +1446,10 @@ static int ecp_mul_comb( mbedtls_ecp_group *grp, mbedtls_ecp_point *R,
 
 cleanup:
 
-    if( T != NULL && ! p_eq_g )
+    /* Free T unless it has been stored in grp. Will be the
```

Comment:
Can you please clarify the second sentence? It isn't clear whether “will be the case” applies to the case “free T” or the case “it has been stored in grp”. Also, we prefer full sentences, e.g. “This will be the case …”.

Reply:
OK. How's that?